ovirt/resource_ovirt_vm.go

```diff
@@ -290,6 +290,14 @@ func resourceOvirtVM() *schema.Resource {
 					string(ovirtsdk4.VMTYPE_HIGH_PERFORMANCE),
 				}, false),
 			},
+			"instance_type_id": {
+				Type:     schema.TypeString,
```

Comment:
Does schema.UUID exists? If not, maybe worth to introduce it.
Better safety for input.
It's a general suggestion for every uuid field that defined here as a string.

Reply:
no they didn't supply that. Although its widely used.